Sources/Billboard/Utilities/Font+iOS15.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Font+iOS15.swift
+//  
+//
+//  Created by Engin Kurutepe on 04.07.23.
+//
+
+import SwiftUI
+
+extension Font {
+    static func compatibleSystem(_ style: TextStyle, design: Design?, weight: Weight?) -> Font {
```

Comment:
A possibility would be the following, with a bit of magic! ;)
More info here: https://www.thomasdurand.fr/ios/swiftui/swiftui-retrocompatibility-with-disfavored-overload/

```suggestion
@available(iOS, deprecated: 16, message: "use iOS 16 system directly!")
extension Font {
    @_disfavoredOverload
    static func system(_ style: TextStyle, design: Design?, weight: Weight?) -> Font {
```

Reply:
Wow never heard of `@_disfavoredOverload`! LGTM @ekurutepe  what do you think?

Reply:
It's documented as an underscored attribute on the Swift repository: https://github.com/apple/swift/blob/main/docs/ReferenceGuides/UnderscoredAttributes.md#_disfavoredoverload

Although it's mostly fine to use it for the final project (where you own the code) ; it might not for a library indeed